### DISCUSSION TITLE
Add index of target download to logs

### DISCUSSION THREAD

**Discussion by rupinderjeet** [Serene-Arc/bulk-downloader-for-reddit#715] [Ideas]
```
[{TIME} - bdfr.downloader - INFO] - Downloaded submission {POSTID} from {SUBREDDIT} ({INDEX})
[{TIME} - bdfr.archiver - INFO] - Record for entry item {POSTID} written to disk ({INDEX})
```

Can we add index of current download item at the end of log line? It doesn't have to represent an actual index of some sort. Just the index of item in current download of posts, this will help in estimating the end time.

**Comment by Serene-Arc** [comment]
No, we can't. Primarily because there's no way to know how many submissions are left without forcefully filling the subreddit lists, which won't be done in 'normal' operation, and that this problem will be solved with the `--progress-bar` option that is currently being developed by a contributor.

**Reply by rupinderjeet** [reply]
I agree. I meant it as counting. So, if downloaded 1, show 1, if downloaded 200, show 200. I know that max is 1000 or less either way. I am not interested in `{downloaded_count/total}`, but in `{downloaded_count}`. If this can't be done, its alright.

Is the progress bar feature for the overall download, or per reddit-post?

**Reply by Serene-Arc** [reply]
For the overall download. You can possibly parse the logfile as it's being written to see how many posts have archived according to a regex, like with grep and the follow functionality. That would be my recommendation.
